src/map/options.ts

```diff
@@ -323,12 +323,17 @@ export class MapOptions extends OptionsGroup {
                 );
 
                 // set width first, since setting position can influence it
-                modalDialog.style.width = `${modalDialog.offsetWidth}px`;
+                modalDialog.style.width = `${modalDialog.offsetWidth / 1.5}px`;
                 // unset margins when using position: fixed
                 modalDialog.style.margin = '0';
                 modalDialog.style.position = 'fixed';
                 modalDialog.style.top = `${top}px`;
-                modalDialog.style.left = `${left}px`;
+
+                if (screen.width < 1400) {
```

Comment:
I changed this now to work in a slightly different way. It works based on `left` as before, but then if the `window` is smaller than 1400, it will clip to the right edge of the screen. I will commit the changes and let me know if that's alright.